web/src/stores/jwt.ts

```diff
@@ -0,0 +1,10 @@
+// import { persist, sessionStorage } from "@macfja/svelte-persistent-store"
```

Comment:
Remove unused import? If we're not gonna make use of sessionstorage, then the default svelte-store should be fine

Reply:
Done!